### PR TITLE
Add volume configuration

### PR DESCRIPTION
**Pull request by oleksiimorozenko**
This patch adds container volume configuration

### CODE REVIEW

**Comment by Shr3ps** [comment]
Thanks for your contribution.

Can we rebase this to the next version of the module to use Terraform 1.3? or do you need this with older terraform/azurerm version?

File: variables.tf [hunk 1]
```diff
@@ -47,6 +47,17 @@ map(
       port     = number
       protocol = string
     }))
+
+    volume = maps(object({
```
Comment (rossifumax) [inline variables.tf]:
```suggestion
    volume = map(object({
```

Comment (rossifumax) [reply variables.tf]: Can you format this file with `terraform fmt` please ?

File: r-aci.tf [hunk 2]
```diff
@@ -36,6 +36,7 @@ resource "azurerm_container_group" "aci" {
       secure_environment_variables = lookup(container.value, "secure_environment_variables", null)
       commands                     = lookup(container.value, "commands", null)
 
+
```
Comment (rossifumax) [inline r-aci.tf]:
```suggestion
```
This is not needed

**Review by rossifumax — CHANGES_REQUESTED** [review]

File: variables.tf (hunk 1 above)
Comment (oleksiimorozenko) [reply variables.tf]: It _is_ formatted with `fmt`

**Comment by oleksiimorozenko** [comment]
> Can we rebase this to the next version of the module to use Terraform 1.3? or do you need this with older terraform/azurerm version?

I used Terraform v1.3.0 when patching/testing and applying my configuration without any problem so I think yes, v1.3 looks good, but
```│ Warning: Deprecated attribute
│
│   on .terraform/modules/diagnostics/locals.tf line 8, in locals:
│    8:     try(data.azurerm_monitor_diagnostic_categories.main[0].logs, [])
│
│ The attribute "logs" is deprecated. Refer to the provider documentation for details.
│
│ (and one more similar warning elsewhere)
```
is telling us that there is submodule which needs to be upgraded as Azure 4.x provider comes into scene

**Review by Shr3ps — APPROVED** [review]

**Comment by Shr3ps** [comment]
Released in v7.1.0

Thanks

**Pull request merged**